docs/configuration.rst

```diff
@@ -76,6 +78,16 @@ PostgreSQL database connection is defined like this:
       "constring": "postgresql://user@localhost:5432/database"
     }
 
+Presto
+
+.. code:: json
+
+    {
+      "name": "local",
+      "type": "presto",
+      "constring": "presto://user@localhost:8889/default"
+    }
```

Comment:
What do you think about changing this type to `sqlalchemy`, and mentioning that every database supported by SQLAlchemy should work (but that we only tested it with postgres and presto)?

Reply:
Sounds good